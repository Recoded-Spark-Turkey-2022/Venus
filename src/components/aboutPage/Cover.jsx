import React from 'react';
import { motion } from 'framer-motion';
import AboutUs from '../../assets/aboutUs.png';

function Cover() {
  return (
    <motion.div
      initial={{ scale: 0.5 }}
      whileInView={{ scale: 1 }}
      transition={{ duration: 0.6, ease: 'linear' }}
      viewport={{ once: false, amount: 0.1 }}
      className="flex justify-center"
    >
      <img src={AboutUs} alt="/" className="w-full h-1/2" />
    </motion.div>
  );
}

export default Cover;
