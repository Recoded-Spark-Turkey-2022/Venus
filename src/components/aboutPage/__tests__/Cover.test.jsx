import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from "react-test-renderer";
import Cover from '../Cover';

jest.mock('swiper/react', () => ({
    Swiper: 'Swiper',
    SwiperSlide: 'SwiperSlide'
  }));
  
  jest.mock('swiper', () => ({
    Pagination: 'Pagination'
  }));

  jest.mock('react-i18next', () => ({
    // this mock makes sure any components using the translate hook can use it without a warning being shown
    useTranslation: () => {
      return {
        t: (str) => str
      };
    },
  }));
  
it(`renders correctly` , () => {
    const tree = renderer.create(
    <Router>
    <Cover />
    </Router>);
    expect(tree).toMatchSnapshot();
});