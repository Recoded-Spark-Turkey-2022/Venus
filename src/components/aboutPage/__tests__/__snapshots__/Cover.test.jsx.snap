// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex justify-center"
>
  <img
    alt="/"
    className="w-full h-1/2"
    src="aboutUs.png"
  />
</div>
`;
